docs/how-to/enable-intel-gpu.rst

```diff
@@ -0,0 +1,118 @@
+.. _enable_intel_gpu:
+
+Enable Intel GPUs
+=============================
+
+This guide describes how to configure DSS to utilise the Intel GPUs on your machine. This can be done by enabling the Intel device plugin on your MicroK8s cluster.
+
+Prerequisites
+-------------
+
+* :ref:`DSS is installed <install_DSS_CLI>` and :ref:`initialised <initialise_DSS>`.
+* The `kubectl` snap package is installed.
```

Comment:
Can we add a link to `kubectl` installation or similar?

Reply:
Added.